src/platform/packages/shared/kbn-es-query/src/es_query/helpers/exclude_frozen_query.ts

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import type { estypes } from '@elastic/elasticsearch';
+import { excludeTiersQuery } from './exclude_tiers_query';
+
+export function excludeFrozenQuery(): estypes.QueryDslQueryContainer[] {
```

Comment:
It doesn't look like this `excludeFrozenQuery` or `excludeTiersQuery` are being used... am I missing something?